kyverno-pss-policy-exceptions/README.md

```diff
@@ -0,0 +1,167 @@
+
+Future improvements to the security platform intend to centralize the exceptions and eliminate the need to maintain individual application-level resources, moving us to a Centralized Exception model.
+
+### Workload Cluster Default Implementation
+
+We currently include a default restricted PSP in each workload cluster. This will be replaced by Kyverno PSS policies, enforced in `restricted` mode.
+
+Customers will have great flexibility in how they deliver `PolicyExceptions` to the cluster for any workloads which are not compliant with the `restricted` standard. Examples of possible exception models are included below.
+
+A newly-created workload cluster will contain `PolicyException` resources deployed for our applications, described above.
+
+### Migration Implications
+
+If this approach is accepted, we expect the migration to involve the following effort from Giant Swarm and our customers.
+
+Giant Swarm:
+
+- creation of `PolicyExceptions` for our applications which require them and can't be made compliant
+  - Shield has already made many workloads compliant so they don't require exceptions. Others can still be made compliant.
+  - this work could be done in parallel with removal / gating of PSP resources which will need to be removed in post-v1.25 clusters anyway
+- (Shield) creation of assistive guides. This is already in progress in parallel, but requires agreement on this RFC to be finalized
+
+Customers:
+
+- creation of `PolicyExceptions` for customer workloads
+  - this effort depends significantly on the customer's current model of PSP usage and application behavior
+    - for example, we have customers who centrally control all PSPs and customers who allow teams to control their own. Effort for the second is likely to be higher, also from a future management perspective
+
+Shield's experience with early Kyverno adopters is that organizations have drastically different ability and willingness to enforce security policies.
+We will provide resources based on what we already know from early adopters, what we learn from Giant Swarm's migration, and the unknown needs that we will inevitably learn from customers as they begin the move.
+
+### Migration Path
+
+More detailed information on the migration path will be published after agreement is reached on this RFC. To help understand the path to PSS adoption, the high-level migration process would be:
+
+1. Kyverno and PSS policies (audit) are added to an upcoming release. This could be a new v19 minor.
+    - Result: customers now have information about non-compliant workloads
+1. Customers begin creating `PolicyExceptions` for workloads which require them
+    - Shield provides docs and support
+    - Optionally: customers change individual policies to `enforce` as compliance is achieved
+    - Optionally: customers can elect to completely disable PSPs
```

Comment:
meaning in combination with the option to enforce PSS already in this step they would make their PSP migration before moving to 1.25
assuming the first step includes us being compliant already

I'm currently thinking if this should not be our default route to avoid bringing too many changes into 1.25 (it contains some other breaking changes, too)

Reply:
> meaning in combination with the option to enforce PSS already in this step they would make their PSP migration before moving to 1.25

Correct. My assumption is we will get there before most customers and have 1.25 on MCs, but I don't think we need to push specifically to enable customers to disable PSPs in 1.24.